### ISSUE TITLE
Prevent `git machete github checkout-prs/anno-prs` from overwriting already present branch annotations

### ISSUE THREAD

**Issue by amalota** [VirtusLab/git-machete#717]
Still need to decide whether it's allowed to add text to the current annotation or if it's not allowed to write at all but i would go with the latter - might be useful in https://github.com/VirtusLab/git-machete/issues/293.

**Comment by PawelLipski** [comment]
Already addressed on https://github.com/VirtusLab/git-machete/pull/730 right?

**Comment by amalota** [comment]
> Already addressed on #730 right?

No, #730 prevents from overwriting qualifiers but not the whole annotation.

**Comment by PawelLipski** [comment]
I think the current behavior is viable, let's leave as is
